webpack.config.babel.js

```diff
@@ -90,6 +90,8 @@ const baseStyleRules = (disableExtractToFile = false) =>
 const PROD_CONFIG = {
   'ONFIDO_API_URL': 'https://api.onfido.com',
   'ONFIDO_SDK_URL': 'https://sdk.onfido.com',
+  'ONFIDO_TERMS_URL': 'https://onfido.com/termofuse',
```

Comment:
oops sorry my bad it should be /termsofuse (I realise I mistyped termofuse in the ticket!)